settings.gradle

```diff
@@ -18,11 +18,11 @@ include 'branchLayout:api'
 include 'branchLayout:impl'
 include 'backend:api'
 include 'backend:impl'
-include 'frontend:icons'
 include 'frontend:base'
 include 'frontend:actions'
 include 'frontend:file'
 include 'frontend:graph:api'
 include 'frontend:graph:impl'
 include 'frontend:ui:api'
 include 'frontend:ui:impl'
+include 'frontend:resourcebundles'
```

Comment:
Nit: as per the comment above, pls keep the projects in [topological order](https://pl.wikipedia.org/wiki/Sortowanie_topologiczne)